src/addon/cc-addon-option-form.js

```diff
@@ -0,0 +1,124 @@
+import '../addon/cc-addon-option.js';
+import '../atoms/cc-button.js';
+import '../molecules/cc-block.js';
+import { css, html, LitElement } from 'lit-element';
+import { dispatchCustomEvent } from '../lib/events.js';
+import { i18n } from '../lib/i18n.js';
+
+/**
+ * A component that displays a form of `<cc-addon-option>`.
+ *
+ * * 🎨 default CSS display: `block`
+ * <br>
+ * 🧐 [component's source code on GitHub](https://github.com/CleverCloud/clever-components/blob/master/src/addon/cc-addon-option-form.js)
+ *
+ * ## Details
+ *
+ * * Emits a `submit` event when the form is submited. The emitted event is an object of options state (enabled / disabled).
+ *
+ * ## Type definitions
+ *
+ * ```js
+ * interface Option {
+ *   title: String,
+ *   logo: String,
+ *   description: DOM,
+ *   name: String,
+ *   enabled: Boolean
+ * }
+ *
+ * interface Options {
+ *   optionName1: Boolean,
+ *   optionName2: Boolean,
+ * }
+ * ```
+ *
+ * @prop {String} title - Title of the whole options form.
+ * @prop {Array<Option>} options - List of Option object to render.
+ *
+ * @event {CustomEvent<Options>} cc-addon-option-form:submit - Fires when the form is submitted.
+ *
+ * @slot description - The description of the add-on and available options.
```

Comment:
Do we need a named slot in this case?

Reply:
I felt like it needed a name just to know what it was about because the options will also be rendered in this component.
```
<cc-addon-option-form .options=${options}>
  my html
</cc-addon-option-form>
```

I found that this layout doesn't really describe what's going on. But I can remove it if you prefer to only have named slots when needed.

Reply:
I wanted to always use the default when we just need one slot but what you're saying makes sense. Let's keep it like that and we'll see in the future...